src/actions/AuthActions.js

```diff
@@ -56,16 +66,17 @@ export function tokenError() {
   }
 }
 
-export function handleLoginSuccess(token) {
+export function handleLoginSuccess(token, formType = '86') {
```

Comment:
I'd recommend creating constant definitions for each of the form types, so that all of the values are centrally defined.